Frameworks/UIKit/XamlControls.h

```diff
@@ -121,6 +121,12 @@ WXCGrid* CreateLabel();
 WXCTextBlock* GetLabelTextBlock(WXCGrid* labelGrid);
 
 ////////////////////////////////////////////////////////////////////////////////////
+// ScrollViewer
+////////////////////////////////////////////////////////////////////////////////////
+WXCGrid* CreateScrollViewer();
+WXCScrollViewer* GetScrollViewer(WXCGrid* labelGrid);
```

Comment:
alternatively, you could return a FrameworkElement since you don't need to know/care that the root element is a grid.

---
In reply to: [106714861](https://github.com/Microsoft/WinObjC/pull/2257#discussion_r106714861) [](ancestors = 106714861,106713268,106712808)